stm32-modules/include/heater-shaker/heater-shaker/motor_task.hpp

```diff
@@ -171,7 +174,9 @@ class MotorTask {
             policy.delay_ticks(MOTOR_START_WAIT_TICKS);
             if ((msg.target_rpm != 0) &&
                 (policy.get_current_rpm() < MOTOR_START_THRESHOLD_RPM)) {
-                error = errors::ErrorCode::MOTOR_UNABLE_TO_MOVE;
+                error = (error == errors::ErrorCode::NO_ERROR)
```

Comment:
why do we have to do this? if it's the underlying driver doing something weird, let's drop a comment to that effect here and add a test.